samples/powerpoint/basics/insert-image.yaml

```diff
@@ -0,0 +1,62 @@
+id: ''
+gist: ''
+source: POWERPOINT
+author: Microsoft
+name: insert-image
+description: Inserts an image to the current slide.
+script:
+  content: |
+    $('#insert').click(run);
+
+    async function run() {
+        var base64EncodedImageStr = getImageAsBase64String();
+
+        await Office.context.document.setSelectedDataAsync(base64EncodedImageStr, {
```

Comment:
You can't await the old style APis, can you?  They're not promises.  I'm kinda surprised it would even compile...